### PR TITLE
Yassine branch: made autocomplete modal.

### PR DESCRIPTION
**Pull request by awsm36z**
Made AutoComplete Modal

### CODE REVIEW

File: Dev/QConnect/src/components/AssignmentEntryComponent.js [hunk 1]
```diff
@@ -3,15 +3,20 @@ import { Modal, Text, TouchableHighlight, View, StyleSheet } from 'react-native'
 import QcActionButton from 'components/QcActionButton'
 import strings from 'config/strings';
 import colors from 'config/colors';
-import AutoSuggest from 'react-native-autosuggest';
+//import AutoSuggest from 'react-native-autosuggest';
```
Comment (zyadelgohary) [inline Dev/QConnect/src/components/AssignmentEntryComponent.js]: Just remove this line

File: Dev/QConnect/src/components/AssignmentEntryComponent.js [hunk 2]
```diff
@@ -24,14 +29,18 @@ export default class AssignmentEntryComponent extends React.Component {
...
                         <Text style={styles.modalTitle}>Enter Assignment</Text>
-                        <AutoSuggest
+                        {/* <AutoSuggest
                             onChangeText={(text) => this.setState({ input: text })}
                             terms={surahNames.map(value => { return value.name })}
                             onItemPress={(value) => this.setState({ input: value })}
                             textInputStyles={{ backgroundColor: colors.lightGrey }}
                             placeholder="eg. Al-Fatihah Ayahs 1-5"
-                        />
+                        /> */}
```
Comment (zyadelgohary) [inline Dev/QConnect/src/components/AssignmentEntryComponent.js]: If this is unused, just remove it

File: Dev/QConnect/src/components/AutoCompleteComponent/InputAutoSuggest.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
+
+import {
+  FlatList, View, TextInput, StyleSheet,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import * as _ from 'lodash';
+// import { TextInput } from 'react-native-ui-lib';
```
Comment (zyadelgohary) [inline Dev/QConnect/src/components/AutoCompleteComponent/InputAutoSuggest.js]: Remove this line

File: Dev/QConnect/src/components/AssignmentEntryComponent.js (hunk 1 above)
Comment (awsm36z) [reply Dev/QConnect/src/components/AssignmentEntryComponent.js]: Ok, I’ll tell you when I finish that

File: Dev/QConnect/src/components/AutoCompleteComponent/InputAutoSuggest.js (hunk 3 above)
Comment (awsm36z) [reply Dev/QConnect/src/components/AutoCompleteComponent/InputAutoSuggest.js]: Done

**Pull request merged**